repose-aggregator/components/filters/saml-policy-translation-filter/src/main/resources/META-INF/schema/examples/saml-policy.cfg.xml

```diff
@@ -1,9 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <saml-policy xmlns="http://docs.openrepose.org/repose/samlpolicy/v1.0">
-    <keystone-credentials uri="keystone.somewhere.com"
-                          username="aUsername"
-                          password="somePassword"
-                          connection-pool-id="default"/>
-    <policy-cache ttl="300" atom-feed-id="identity-policy"/>
+    <policy-bypass-issuers>
+        <issuer>http://foo.bar</issuer>
+        <issuer>http://notmyfirst.rodeo</issuer>
+    </policy-bypass-issuers>
+    <policy-acquisition>
+        <keystone-credentials uri="http://keystone.somewhere.com"
+                              username="aUsername"
+                              password="somePassword"
+                              connection-pool-id="default"/>
+        <policy-endpoint uri="http://keystone.somewhere.com"
+                         connection-pool-id="different"/>
+        <cache ttl="300" atom-feed-id="identity-policy"/>
+    </policy-acquisition>
+    <signature-credentials keystore-filename="thing.jks"
+                           keystore-password="banana"
+                           key-name="thingy"
+                           key-password="phone"/>
 </saml-policy>
```

Comment:
We might want to stick a copy of this config file in the appropriate artifacts directory before we forget.